defaults/main.yml

```diff
@@ -6,6 +6,7 @@ borgbackup_ssh_key: "~{{ borgbackup_client_user }}/.ssh/id_borg_rsa"
 
 borgbackup_binary: borg
 borgbackup_install_from_pkg: false
+borgbackup_install_epel: true
```

Comment:
Since `install_from_pkg` is default to `false`, I think this should be as well.

Could you add a section to the README about how to install via EPEL (change these two variables to `true`)?

Reply:
I can flip that to false, but the task file for managing EPEL won't be included unless `borgbackujp_install_from_pkg` is true, so it wouldn't end up installing EPEL extraneously.